packages/babel-preset-fbjs/plugins/__tests__/inline-requires-test.js

```diff
@@ -53,6 +53,56 @@ pluginTester({
       snapshot: false,
     },
 
+    'ensures that the inlined require still points to the global require function': {
```

Comment:
I probably haven't formatted the files correctly in my patch. Could you make sure that the file conforms to the formatting guidelines (or run prettier?).

Reply:
I disabled max-len for this entire file.